terraform-provider-constellation/internal/provider/image_data_source.go

```diff
@@ -66,14 +69,14 @@ func (d *ImageDataSource) Metadata(_ context.Context, req datasource.MetadataReq
 // Schema returns the schema for the image data source.
 func (d *ImageDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		Description:         "Data source to retrieve the Constellation OS image reference for a given CSP and Attestation Variant.",
-		MarkdownDescription: "Data source to retrieve the Constellation OS image reference for a given CSP and Attestation Variant.",
+		Description:         "The data source to resolve the CSP-specific OS image reference for a given version and attestation variant.",
```

Comment:
Same as for the attestation data source, we might want to add a mention of the lock-step versioning behaviour if no explicit version is set to the description here.

Reply:
added